ibm/mas_devops/roles/cp4d_db2wh_manage_hack/tasks/main.yml

```diff
@@ -2,13 +2,39 @@
 # Should should be part of the manage operator, but is not so we have to do it
 # as a seperate step in the install flow for now :(
 
+# 0.1. Check which CPD version has been installed (in case cpd_version is not informed)
+# -----------------------------------------------------------------------------
+- name: Check what CPD version has been installed by looking at the namespaces
+  community.kubernetes.k8s_info:
+    api_version: v1
+    kind: Namespace
+    name: cpd-services
```

Comment:
in cpd v3.5 the namespace cpd was installed to was configurable .. have we removed the ability to configure where it's installed with cpd v4, as this logic will only work if the namespace **has** to be cpd-services, and there is no possibility to configure it otherwise.

Reply:
In cpd4.0 I noticed the vars are loaded and overwrite whatever is in cpd namespace environment var, different from cpd 3.5 (where it uses whatever was sent by users)... my initial thought was to remove the namespace to be configurable for 3.5, but then thouht someone may be relying on it already so I kept it as it is, but did not include the option to do the same for 4.0, keeping the code as it was originally created, loading internal vars, etc

But it does not mean we can introduce the configurable namespaces for 4.0 in future patches if needed (then we will need two envvars, one for cpd, another one for services, etc), not necessarily part of this specific PR though. Makes sense?

Reply:
Okay, given this info I think the most sensible thing to do is to remove that variable for 3.5 as well and just have it so it always installs to a set namespace, so we can offer a consistent behaviour across the two versions.  

I see minimal value in customising the install directory of cp4d, we've never used it ourselves; and keeping it so that the env var works sometimes and not others depending on what version of cp4d is being installed is not good given part of the job of the collection is to allow users to not have to think about what version of cp4d to install as we will work it out from the mas_channel etc.